dts/msc-sm2-imx6x.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
This is a step back in maintainability, but I do think we should indeed revert this back to the MSC supplied version, and overrule the settings with our custom and more maintainable settings in our own DTS files.